src/test/java/mate/academy/dao/RoleDaoTest.java

```diff
@@ -0,0 +1,78 @@
+package mate.academy.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Optional;
+import mate.academy.dao.impl.RoleDaoImpl;
+import mate.academy.model.Role;
+import org.hibernate.SessionFactory;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+public class RoleDaoTest extends AbstractTest {
+    private RoleDao roleDao;
+    private Role userRole;
+    private Role adminRole;
+
+    @Override
+    protected Class<?>[] entities() {
+        return new Class[] {Role.class};
+    }
+
+    @BeforeEach
+    void beforeEach() {
+        userRole = new Role();
```

Comment:
You can initialize object in @BeforeClass once and then in @BeforeEach just set new values